newsletter-june-2023.rst

```diff
@@ -0,0 +1,80 @@
+.. post:: June 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+   :category: Newsletter
+
+Read the Docs newsletter - June 2023
+====================================
+
+News and updates
+----------------
+
+- ⚠️ It will be required to use a ``.readthedocs.yaml`` configuration file.
+  It's highly encouraged to add one already today.
+  Read more about this change in :doc:`/migrate-configuration-v2`.
+- ✅️ Visiting ``/en/`` on a documentation project now redirects to ``/en/latest/``.
+- 🎥️ All the talks from `Write the Docs Portland 2023 <https://www.writethedocs.org/conf/portland/2023/>`__ can now be watched for free:
+  `Open the new playlist on Youtube <https://www.youtube.com/watch?v=EZJ0mk9Jj3s&list=PLZAeFn6dfHpneQPsDWa4OmEpgW4pNiaZ2>`__.
+- 🐛️ Builds with multiline commands suffered from `a few bugs <https://github.com/readthedocs/readthedocs.org/issues/10103>`__ that are hopefully all fixed now.
+  Thanks to everyone who helped out!
+- 🐛️ `sphinx-rtd-theme 1.2.1 <https://sphinx-rtd-theme.readthedocs.io/en/stable/changelog.html>`__ has been released with an important bug fix that caused jQuery to not load in certain projects.
+- 🔒️ We updated custom domain certificates to 1.2 as the *minimum* TLS version.
+- 🔒️ A high-severity vulnerability has been fixed:
+  `Write access to projects via API V2 (/api/v2/project/* endpoints) for any logged-in user <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-rqfv-8rrx-prmh>`__.
+
+Upcoming features
+-----------------
+
+- 🚢️ We have concluded work on the first phase of our new `readthedocs-client <https://github.com/readthedocs/readthedocs-client>`_.
+  The client will be a single library for all the ways Read the Docs enhances documentation from having a flyout menu
+  to link to other versions, translations, or subprojects, to showing version warnings for folks browsing outdated docs
+  to running ads on community projects.
+  There will be integration points and APIs to customize all of these features in the new client.
+
+  We will start to use it on a few selected projects.
+  If you are interested in having it enabled on your project,
+  please `reach out`_.
```

Comment:
Trying again to stir a bit of hype around this :)

Can you suggest changes that remove any over-promising? And can we have another nice screenshot here? (this newsletter would like a screenshot)

CC: @humitos @agjohnson I promised to bring this up again after it was postponed last time :innocent:  :wave:

Reply:
This is not 100% accurate. The new client is enabled automatically for projects using `build.commands` now and it's not feature complete (no flyout, for example). However, we need to work around a CF quota limitation.

In other words, I think this is not ready to be announced here yet.

Reply:
It's great to have something in the upcoming horizon that makes readers know that we're doing cool things. I'll try to find something else for the roadmap :)